### PR TITLE
Create alden.is-a.dev for my best friend's website

### PR DESCRIPTION
**Pull request by ghost**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://aldenpbm.github.io

### CODE REVIEW

File: domains/alden.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "repo": "https://github.com/Schmeichel-web/aldenpbm.github.io",
+    "owner": {
+        "username":"Schmeichel-web",
+        "manager": "cupglassdev",
+        "email": "",
+        "discord": "footballfan0891",
+        "note":"this is my friend's website, because he dosent really understands about these thing, im managing it"
+    },
+    "record": {
+        "CNAME": "aldenpbm.github.io"
```
Comment (DEV-DIBSTER) [inline domains/alden.json:12]: I'm getting a 404 page, is there any content there?

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/alden.json (hunk 1 above)
Comment (cupglassDEV) [reply domains/alden.json:12]: Yes, there is a content in there

Comment (cupglassDEV) [reply domains/alden.json:12]: https://github.com/Schmeichel-web/aldenpbm.github.io

**Comment by cupglassDEV** [comment]
Hey, sorry for the problem. My friend forgot to deploy the pages

image

Now its working for now

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**